### PR TITLE
Add GitHub-Button : Added the GitHub button

### PR DESCRIPTION
**Pull request by Shreeom03**
Added the Button referencing to GitHub repository and using the poll image as button image. Also used Tailwind CSS used to design the webpage.

Please go through with the code and the webpage and please let me know if any changes required.

### CODE REVIEW

File: index.html
Comment (Ramzi-Abidi) [inline index.html:1]: Please try to solve one problem at a time.

**Comment by Ramzi-Abidi** [comment]
Please try to solve one single issue at the time.

Tailwind will be added in the future since the project is quite simple now.

Please include a screenshot of the work you've done. And thanks for your time!

**Review by Ramzi-Abidi — CHANGES_REQUESTED** [review]
Please make the necessary adjustments according to my previous comment, and once done, submit it for merging. Thank you.

**Comment by Shreeom03** [comment]
Then how should i style the button ?
Can i use css ? or you just want changes in HTML file ?

**Comment by Ramzi-Abidi** [comment]
> Then how should i style the button ? Can i use css ? or you just want changes in HTML file ?

Yes use vanilla CSS

**Pull request closed without merging**